include/moveit_visual_tools/moveit_visual_tools.h

```diff
@@ -358,6 +365,10 @@ class MoveItVisualTools : public rviz_visual_tools::RvizVisualTools
    * \param color to display the collision object with
    * \return true on sucess
    **/
+  moveit_msgs::CollisionObject createCollisionObject(const Eigen::Isometry3d& pose, double width, double depth,
```

Comment:
same comments:
rename function to include Cuboid

Reply:
done